### DISCUSSION TITLE
authentication to RDS aurora serverless

### DISCUSSION THREAD

**Discussion by OElGebaly** [gravitational/teleport#7459] [Q&A]
any workarounds for authentication with RDS aurora serverless databases ?
there is a blocker for authentication to aurora serverless via teleport as the database is not allowing authentication via AWS IAM policies, but is there any other workarounds that can be applied to enable authentication ?

**Comment by webvictim** [comment]
@r0mant will know for sure, but I believe that currently there is no workaround to handle authentication for Aurora serverless databases via Teleport Database access. This is because they don't allow IAM authentication as required.
